pd/src/genesis.rs

```diff
@@ -47,29 +48,19 @@ impl Allocation {
     }
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ValidatorPower {
     pub validator: Validator,
     pub power: tendermint::vote::Power,
 }
 
 /// The application state at genesis.
-#[derive(Deserialize, Serialize, Debug, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
 pub struct AppState {
     /// The number of blocks in each epoch.
-    pub epoch_duration: u64,
+    pub chain_params: ChainParams,
```

Comment:
outdated doc comment?

Reply:
yup, looks like an automerge caused it to get out of date, fixed now